scripts/android-build-release.conf

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Android build confguration
+#
+# Note: This is a simple configuration to build all
+#       architectures in one rush.
+#       Since android 64 bit support was introduced with NDK API 21
+#       this is the minimal common denominator.
+#       If you require support for older NDK API levels,
+#       create seperate configurations for each NDK API level
+#       and architecture you want to support.
+WITH_JPEG=0
+WITH_OPENH264=1
+WITH_OPENSSL=1
+BUILD_DEPS=1
+DEPS_ONLY=0
+NDK_TARGET=26
```

Comment:
While NDK 26 is nicer for debugging, if you want more wide availability 21 would most likely be the best option. Don´t know current usage stats for f-droid and what the sweetspot is there.

Reply:
Done. Sorry, forgot when copy-pasting